src/main/resources/standardTestCases/sub_workflow_decls/sub_workflow_decls_import.wdl

```diff
@@ -0,0 +1,43 @@
+task firstTask {
+  command {
+    echo foo
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+  output {
+    String out = read_string(stdout())
+  }
+}
+
+task secondTask {
+  String relay
+  command {
+    echo ${relay}
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+  output {
+    String out = read_string(stdout())
+  }
+}
+
+workflow sub_decls {
+  # A subworkflow declaration that requires an input.
+  String uninitialized
+  # A subworkflow declaration that's initialized but which we expect to be overridden.
+  String wantsOverride = "don't want to see this"
```

Comment:
We want to test that we get what we expect in these situations:
- [X] A subworkflow input that needs to be supplied
- [X] A subworkflow input with a default which is overridden
- [ ] A subworkflow declaration with a default that depends on another input which has no default
- [ ] A subworkflow declaration with a default that depends on another input which is overridden
- [ ] An overridden subworkflow declaration with a default that depends on another input which has no default
- [ ] An overridden subworkflow declaration with a default that depends on another input which was overridden
- [X] A subworkflow declaration which depends on a task output
- [ ] An optional subworkflow input that's not supplied
- [ ] An optional subworkflow input that's got a default
- [ ] An optional subworkflow input that's got a default but is overridden

Reply:
29 hotfix with these changes can handle all but uninitialized optionals which unfortunately appear to be the Achilles' heel of subworkflows.